recipe/meta.yaml

```diff
@@ -29,16 +29,17 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - sqlite
+    - libtiff
+    - libcurl
   run:
     - sqlite
+    - libtiff
+    - libcurl
   run_constrained:
     - proj4 ==999999999999
 
 test:
   commands:
-    # Check if NAD27/NAD83 datum shifting is present if `conus` is present.
-    - test -f $PREFIX/share/proj/conus  # [not win]
-    - if not exist %LIBRARY_PREFIX%\\share\\proj\\conus exit 1  # [win]
```

Comment:
Do we want to replace these tests?  If so, what with?

Reply:
We probably don't need that. These tests were from before we packaged the datum grids and they probably belong in those feedstock instead of here.